File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/03.Conditional-Statements-Advanced/02-exercise/01.cinema.js
function cinema(input) {

    const premierePrice = 12.00;
    const normalPrice = 7.50;
    const discountPrice = 5.00;

    const projection = input[0];
    const rowCount = Number(input[1]);
    const columnCount = Number(input[2]);

    const seatCount = rowCount * columnCount;
    let income = 0;

    switch (projection) {
        case "Premiere": income = seatCount * premierePrice; break;
        case "Normal": income = seatCount * normalPrice; break;
        case "Discount": income = seatCount * discountPrice; break;
    }

    console.log(`${income.toFixed(2)} leva`);
}
cinema(["Discount", "12", "30"]);

File: Python_Basics/Homeworks-And-Labs/03.conditional_statements_advanced/02.exercise/01.cinema.py
premierePrice = 12.00
normalPrice = 7.50
discountPrice = 5.00

projection = input()
rowCount = int(input())
columnCount = int(input())

seatCount = rowCount * columnCount
income = 0

if projection == 'Premiere':
    income = premierePrice
elif projection == 'Normal':
    income = normalPrice
elif projection == 'Discount':
    income = discountPrice

income *= seatCount 

print(f'{income:.2f} leva')